pom.xml

```diff
@@ -38,12 +38,12 @@
 
         <kafka.version>3.0.1</kafka.version>
         <basescala.version>2.13</basescala.version>
-        <junit.version>5.7.1</junit.version>
-        <mockito.version>1.10.19</mockito.version>
+        <junit.version>5.8.2</junit.version>
+        <mockito.version>4.6.1</mockito.version>
         <slf4j.version>1.7.30</slf4j.version>
         <guava.version>30.0-jre</guava.version>
         <zookeeper.version>3.6.3</zookeeper.version>
-        <kafka-junit.version>4.2.0</kafka-junit.version>
+        <kafka-junit.version>4.2.2</kafka-junit.version>
```

Comment:
The dependency error comes from the `kafka-junit.version` should be 4.2.1 or below. FYI

Reply:
Yikes a lot of change in patch release. Given this was just meant to be a simple background task I've reverted to 4.2.1 will leave unravelling the dependency tree for another day.